akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -5,25 +5,43 @@
 package akka.remote.transport.netty
 
 import java.io.{ FileNotFoundException, IOException }
-import java.security._
+import java.nio.file.{ Files, Paths }
+import java.security.{ KeyStore, _ }
 import java.util.concurrent.atomic.AtomicReference
-import javax.net.ssl.{ KeyManagerFactory, SSLContext, TrustManagerFactory }
+import javax.net.ssl._
 
+import akka.actor.ReflectiveDynamicAccess
 import akka.event.{ LogMarker, MarkerLoggingAdapter }
 import akka.japi.Util._
 import akka.remote.RemoteTransportException
 import akka.remote.security.provider.AkkaProvider
+import akka.util.Collections.EmptyImmutableSeq
 import com.typesafe.config.Config
 import org.jboss.netty.handler.ssl.SslHandler
 
 import scala.annotation.tailrec
-import scala.util.Try
-import java.nio.file.Files
-import java.nio.file.Paths
+import scala.util.{ Failure, Success, Try }
 
 /**
  * INTERNAL API
  */
+private[akka] object KeyStoreHelper {
```

Comment:
Needs also `@InternalApi` annotation.

Reply:
The NettySSLSupport class at the bottom has internal api doc but no annotation. Should I ad an annotation there as well?
```
/**
 * INTERNAL API
 *
 * Used for adding SSL support to Netty pipeline
 */
private[akka] object NettySSLSupport {...
```